tensorflow/compiler/xla/metric_table_report.cc

```diff
@@ -80,9 +80,11 @@ void MetricTableReport::WriteReportToInfoLog(double expected_metric_sum) {
 
   int64 pos = 0;
   const string report = MakeReport(expected_metric_sum);
-  while (pos < report.size()) {
+  const int report_size = report.size();
+  while (pos < report_size) {
     int64 end_of_line = report.find('\n', pos);
-    if (end_of_line == string::npos) {
+    int64 _npos = string::npos;
```

Comment:
`const` here. In general, if we create a new variable to eliminate a type cast. we should make that variable `const`. Here, and throughout the PRs.